File: prjonlineshopping/src/app/models/FilterView.model.ts
export class FilterView {
    constructor() {
        this.SortBy = null;
        this.Search = null;
        this.MaxPrice = 0;
        this.MinPrice = 0;
    }
    SortBy: string;
    Search: string;
    MaxPrice: number;
    MinPrice: number;
}

File: OnlineShopping/OnlineShopping/Models/FilterViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OnlineShopping.Models
{
    public class FilterViewModel
    {
        public string SortBy { get; set; }
        public string Search { get; set; }
        public int MaxPrice { get; set; }
        public int MinPrice { get; set; }
    }
}